Source/engine/dx.cpp

```diff
@@ -114,6 +114,8 @@ void dx_cleanup()
 	RendererTextureSurface = nullptr;
 #ifndef USE_SDL1
 	texture = nullptr;
+	FreeVirtualGamepadGFX();
+	FreeVirtualGamepadTextures();

```

Comment:
Perhaps it's better to do this in `FreeGameMem()`, where we already call `FreeVirtualGamepadGFX()`?

Reply:
Good call, since we're initializing the graphics on level load.

Reply:
Ahh, wait. You have a note here that says it has to be called from the main thread. `DoLoad()` calls `FreeGameMem()` from the loading thread.

Reply:
Ah, right, of course

In that case, let's only call `FreeVirtualGamepadTextures` here, because `FreeGameMem` already calls the GFX one?
We might need to eventually restructure this stuff.